CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # AWS Mobile SDK for iOS CHANGELOG
 
+## 2.13.0
+
+### Misc. Updates
+
+- **Breaking Change** Updated nullability flag for the `AWSSignatureSignerUtility`, `AWSSignatureV4Signer`, `AWSSignatureV2Signer` and `AWSS3ChunkedEncodingInputStream` classes. If you are using any of these classes in `swift` code base you might want to add the right specifier for the return values. 
```

Comment:
s/flag/flags/

"If you are using any..." -> "Methods in these classes will have new signatures in Swift code, and may require code changes in your app to consume."

Reply:
Updated

Reply:
The default implementation in the Swift SDK generated by AWSAPIGateway is causing a crash in the line where `AWSSignatureV4Signer` class is being called. Can you suggest a solution for the breakages in the below method, Thank You.

`
init(configuration: AWSServiceConfiguration) {
	    super.init()

	    self.configuration = configuration.copy() as! AWSServiceConfiguration
	    var URLString: String = "https://xh9ws4nwu8.execute-api.ap-south-1.amazonaws.com/develop"
	    if URLString.hasSuffix("/") {
            URLString.remove(at: URLString.index(before: URLString.endIndex))
	    }
	    self.configuration.endpoint = AWSEndpoint(region: configuration.regionType, service: .APIGateway, url: URL(string: URLString))
	    let signer: AWSSignatureV4Signer = AWSSignatureV4Signer(credentialsProvider: configuration.credentialsProvider, endpoint: self.configuration.endpoint)
	    if let endpoint = self.configuration.endpoint {
	    	self.configuration.baseURL = endpoint.url
	    }
	    self.configuration.requestInterceptors = [AWSNetworkingRequestInterceptor(), signer]
	}
`